data_extractor/item.py

```diff
@@ -155,25 +158,29 @@ def getter(self: AbstractSimpleExtractor, name: str) -> Any:
                 return getattr(duplicated.extractor, name)
             return super(type(self), self).__getattribute__(name)
 
-        def setter(self: AbstractSimpleExtractor, name: str, value: Any) -> Any:
-            if hasattr(duplicated.extractor, name):
-                return setattr(duplicated.extractor, name, value)
-            return super(type(self), self).__setattr__(name, value)
-
         classname = f"{type(duplicated).__name__}Simplified"
         base = AbstractSimpleExtractor
         if duplicated.extractor is not None:
             base = type(duplicated.extractor)
 
-        return type(
+        new_cls = type(
```

Comment:
Why does the coverage say that this line of code can not jump to the function exit?

Reply:
`lambda self: None` was not being invoked